packages/dataviews/src/components/dataviews-filters/filter.tsx

```diff
@@ -152,6 +156,58 @@ const FilterText = ( {
 		);
 	}
 
+	if ( filterInView?.operator === OPERATOR_LESS_THAN ) {
```

Comment:
I wonder why we don't use a `switch ( filterInView?.operator ) {` statement. Just a comment, I think we should change that

Reply:
I'm fine with either way. If everyone is fine with this, I can change it. 🙂

Reply:
I'd say not for the moment.